containers/documentation.docker/build_documentation

```diff
@@ -20,22 +20,28 @@
 ## get script dir
 script_dir="$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")"
 
-# determine if this is run in Cygwin under Windows
-is_cygwin=false
-if [[ "$(uname -o || true)" == "Cygwin" ]]; then
-    is_cygwin=true
-fi
-
 # Convert to Windows path in Cygwin, i.e. '/cygwin/c/...' to 'C:\...'
+# will wrap multiple path in one go
 expand_cygpath() {
-    local any_path="$1"
-    if [[ -n $any_path ]] && $is_cygwin; then
-        echo "$(cygpath -w $any_path)"
-    else
-        echo "${any_path}"
-    fi
+    local path
+    local out=()
+    for path in "$@"; do
+        local matchre='/cygwin/'
+        if type cygpath 2>/dev/null; then
```

Comment:
Does this catch empty args?

```
$ cygpath -w ""
cygpath: can't convert empty path
```

Even with no cache path it should never be empty ("fake cache" is passed in that case), but there is a check in the cache code that tests for empty string as well.

Reply:
It is meant to take arbitrary arguments including an empty string, that way all arguments `fs paths` or not can be handled with one call.

Reply:
The RE should make sure that there is no empty string passed to the cypath command.